telemetry/main.schema.json

```diff
@@ -267,7 +267,7 @@
               "type": "boolean"
             },
             "isDefaultBrowser": {
-              "type": "boolean"
+              "type": ["boolean", "null"]
```

Comment:
Should we make this change to the `crash` schema as well? (aside: we really should factor out the common ping structure, this repetition is no good).